### PR TITLE
zero 1.18.0

### PR DESCRIPTION
**Pull request by 2075**
Hi there!
Please find a new type def PR, now based on an external type defs repo.
Thanks in advance,
Marco

### CODE REVIEW

File: packages/apps-config/package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "@sora-substrate/type-definitions": "1.4.1",
     "@subsocial/types": "0.5.9-dev.2",
     "@zeitgeistpm/type-defs": "0.2.5",
+    "@zeroio/type-definitions": "^0.0.5",
```
Comment (jacogr) [inline packages/apps-config/package.json:29]:
```suggestion
    "@zeroio/type-definitions": "0.0.5",
```

We have had some issues with `^` dependencies - a number of projects are actually using `@polkadot/apps-config` as stadnd-alone.

**Review by jacogr — COMMENTED** [review]
Just needs a linting fix and range adjustment.

**Review by jacogr — COMMENTED** [review]
Please fix the conflicts/linting to make CI pass.

**Comment by 2075** [comment]
allright, i'm on it :D

**Comment by 2075** [comment]
that escalated quickly

**Pull request closed without merging**